src/test/java/com/google/api/codegen/gapic/testdata/nodejs/nodejs_library.baseline

```diff
@@ -546,28 +579,36 @@ client.getBigBook({name: formattedName})
 // FIXME: Insert here code to prepare the request fields, make the call, process the response.
 
 /*
-// const shelf = 'Novel';
-const formattedName = client.bookPath(shelf, 'War and Peace');
-
-// Handle the operation using the promise pattern.
-client.getBigBook({name: formattedName})
-  .then(responses => {
-    const [operation, initialApiResponse] = responses;
-
-    // Operation#promise starts polling for the completion of the LRO.
-    return operation.promise();
-  })
-  .then(responses => {
-    const result = responses[0];
-    const metadata = responses[1];
-    const finalApiResponse = responses[2];
-
-    console.log(`name: {$result.name}`);
-    console.log(`author: {$result.author}`);
-  })
-  .catch(err => {
-    console.error(err);
-  });
+async function sampleGetBigBook(shelf) {
```

Comment:
so, if we make this async (ignoring Node v6), let's keep going that way and make this fully async-await. Instead of using promises, we can do

```js
async function sampleGetBigBook(shelf) {
  ...
  const [operation, initialApiResponse] = await client.getBigBook({name: formattedName});
  const [result, metadata, finalApiResponse] = await operation.promise();
  ...
}
```

Or if we don't do that, we don't need `async` at all.

Reply:
@alexander-fenster I didn't have a clue what that `async` meant... I mainly copied from the [manual samples](https://github.com/googleapis/nodejs-speech/blob/7528e17bfa3ba476e3c51f0c6e7c39a8f4378c0d/samples/recognize.v1p1beta1.js#L26). But now I get it because the manual samples have `await` calls. I'll remove the `async` for now.